tests.go

```diff
@@ -342,18 +341,24 @@ func deleteRFML(c cliContext) error {
 	return nil
 }
 
-// uploadRFML is a wrapper around test creating/updating functions
-func uploadRFML(c cliContext, api rfmlAPI) error {
+// uploadTests is a wrapper around test creating/updating functions
+func uploadTests(c cliContext, api rfAPI) error {
 	if c.Bool("synchronous-upload") {
 		rfmlUploadConcurrency = 1
 	}
+
 	if path := c.Args().First(); path != "" {
-		err := uploadSingleRFMLFile(path)
+		var err error
+
+    err = uploadSingleRFMLFile(path)
```

Comment:
Weird formatting, or is it GH's diff doing something weird?

Reply:
Why not
```suggestion
err := uploadSingleRFMLFile(path)
```
?

Reply:
Thanks that's a leftover from when there were to options and I needed to declare the `var`, before setting it in an `if/else`.

Will change.